fe/fe-core/src/main/java/com/starrocks/sql/analyzer/AggregationAnalyzer.java

```diff
@@ -318,5 +319,10 @@ public Boolean visitTimestampArithmeticExpr(TimestampArithmeticExpr node, Void c
         public Boolean visitCloneExpr(CloneExpr node, Void context) {
             return visit(node.getChild(0));
         }
+
+        @Override
+        public Boolean visitDictQueryExpr(DictQueryExpr node, Void context) {
+            return node.getChildren().stream().allMatch(this::visit);
+        }
```

Comment:
it wouldn't contains aggregation?

Reply:
can contains agg function, same as regular scalar function.